Form/Type/CategorySelectorType.php

```diff
@@ -65,7 +65,7 @@ public function configureOptions(OptionsResolver $resolver)
                 'context' => null,
                 'category' => null,
                 'choice_list' => function (Options $opts, $previousValue) use ($that) {
-                    return new SimpleChoiceList($that->getChoices($opts));
+                    return new ArrayChoiceList($that->getChoices($opts));
```

Comment:
Yes, there is no issue. This code is here for BC with <SF2.7, and will be removed in next major.